cli-files/go-ls/cmd/root_test.go

```diff
@@ -0,0 +1,90 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestGolsCmdFunc(t *testing.T) {
+	t.Run("No path provided", func(t *testing.T) {
+		r, w, _ := os.Pipe()
+		os.Stdout = w
+
+		cmd := golsCmd
```

Comment:
What's this for?

Reply:
I think I changed how the cmd was being declared and this is a leftover (it was a function returning CobraCmd)